compiler-rt/test/msan/lit.site.cfg.py.in

```diff
@@ -6,6 +6,7 @@ config.target_cflags = "@MSAN_TEST_TARGET_CFLAGS@"
 config.target_arch = "@MSAN_TEST_TARGET_ARCH@"
 config.use_lld = @MSAN_TEST_USE_LLD@
 config.use_thinlto = @MSAN_TEST_USE_THINLTO@
+config.cmake_sysroot = "@CMAKE_SYSROOT@"
```

Comment:
This is just a nit, but I'd suggest calling it just `sysroot`, the fact that it comes from `CMake` isn't relevant.
```suggestion
config.sysroot = "@CMAKE_SYSROOT@"
```